src/rosdistro/freeze_source.py

```diff
@@ -82,12 +82,28 @@ def freeze_distribution_sources(dist, release_version=False, release_tag=False,
         print("")
 
 
-def _worker(work_queue):
+# Get the repo commit information
+def _getRepoInfo(source_repo):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#function-names

Reply:
Darn I even ran pep8 against it.  Thanks I've changed it to the standard.